omnibridge/flows/graph_flow.py

```diff
@@ -0,0 +1,73 @@
+from typing import List, Dict, Any, Union
+
+from omnibridge.model_entities.models_io.base_model_io import ModelIO, TextualIO
+from omnibridge.model_entities.models_io.flow_io import FlowIO
+from omnibridge.saved_data.json_data_manager import JsonConvertable
+from omnibridge.wrappers.wrapper_instances.type_name_to_wrapper import ModelLoader
+from omnibridge.wrappers.wrapper_interfaces.model_wrapper import ModelWrapper
+
+
+class Node(JsonConvertable):
+    def __init__(self, model: ModelWrapper, instruction: str, next_nodes: List[ModelWrapper]):
+        self.next_nodes = next_nodes
+        self.model = model
+        self.instruction = instruction
+
+    def process(self, model_input: ModelIO) -> ModelIO:
+        assert isinstance(model_input, TextualIO)
+        model_input = TextualIO(model_input.get_text() + "\n" + self.instruction)
```

Comment:
just a suggestion, f-strings are the standard in Python + slight performance advantage over concatenation.
```suggestion
        model_input = TextualIO(f'{model_input.get_text()}\n{self.instruction}')
```